src/test/java/au/org/emii/MyIT.java

```diff
@@ -11,8 +11,11 @@
 import java.util.Properties;
```

Comment:
Yes definitely. Same for the units. Let me address it in the next set of changes otherwise I'm going to get lost in rebasing hell, with other file renaming changes.